packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.deprecated.text.tsx

```diff
@@ -0,0 +1,48 @@
+/* tslint:disable-next-line:no-unused-variable */
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+import { setRTL } from '../../../Utilities';
+import { PersonaCoin } from './PersonaCoin';
+
+const testImage1x1 = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAAC0lEQVQImWP4DwQACfsD/eNV8pwAAAAASUVORK5CYII=';
+
+// Prevent warn deprecations from failing test
+const Utilities = require('@uifabric/utilities/lib/warn');
+
+describe('PersonaCoin', () => {
+  beforeEach(() => {
+    setRTL(false);
+  });
+
+  beforeAll(() => {
+    Utilities.warnDeprecations = jest.fn().mockImplementation(() => { /** no impl **/ });
```

Comment:
in my testing, commenting line 17-23 doesn't cause the test to fail..

Reply:
So the problem is you called the file `.text.tsx`

Reply:
Nice catch, that caught a backwards compatibility bug too. Shows the tests are working as intended 😄